apps/meteor/package.json

```diff
@@ -27,7 +27,7 @@
 		"eslint:fix": "eslint --ext .js,.jsx,.ts,.tsx . --fix",
 		"stylelint": "stylelint \"app/**/*.css\" \"client/**/*.css\" \"app/**/*.less\" \"client/**/*.less\" \"ee/**/*.less\"",
 		"stylelint:fix": "stylelint --fix \"app/**/*.css\" \"client/**/*.css\" \"app/**/*.less\" \"client/**/*.less\" \"ee/**/*.less\"",
-		"typecheck": "cross-env NODE_OPTIONS=\"--max-old-space-size=4092\" tsc --noEmit --skipLibCheck",
+		"typecheck": "cross-env NODE_OPTIONS=\"--max-old-space-size=5120\" tsc --noEmit --skipLibCheck",
```

Comment:
I think it is because I keep getting heap exceptions for 4GB. Checkout the previous comment here for more info ->https://github.com/RocketChat/Rocket.Chat/pull/25344#discussion_r866737756